frontend/packages/console-shared/src/components/pod/PodRingSet.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { K8sResourceKind, K8sKind } from '@console/internal/module/k8s';
+import { Split, SplitItem, Bullseye } from '@patternfly/react-core';
+import { LongArrowAltRightIcon } from '@patternfly/react-icons';
+import { PodRCData } from '../../types';
+import { getPodData } from '../../utils';
+import PodRing from './PodRing';
+
+interface PodRingSetProps {
+  podData: PodRCData;
+  obj: K8sResourceKind;
+  resourceKind: K8sKind;
+  path: string;
+  impersonate?: string;
+}
+
+const PodRingSet: React.FC<PodRingSetProps> = ({ podData, resourceKind, obj, path }) => {
+  const { inProgressDeploymentData, completedDeploymentData } = getPodData(
+    obj,
+    podData.pods,
+    podData.current,
+    podData.previous,
+    podData.isRollingOut,
+  );
+  const current = podData.current && podData.current.obj;
+  const previous = podData.previous && podData.previous.obj;
+  const progressRC = inProgressDeploymentData && current;
+  const completedRC = !!inProgressDeploymentData && completedDeploymentData ? previous : current;
+  return (
+    <Split gutter="lg">
+      <SplitItem>
+        <PodRing
+          key={inProgressDeploymentData ? 'deploy' : 'notDeploy'}
+          pods={completedDeploymentData}
+          rc={completedRC}
+          resourceKind={resourceKind}
+          obj={obj}
+          path={path}
+          enableScaling={!podData.isRollingOut}
+        />
+      </SplitItem>
+      {inProgressDeploymentData && (
+        <React.Fragment>
+          <SplitItem>
+            <Bullseye>
+              <LongArrowAltRightIcon size="xl" color="#737679" />
```

Comment:
Please user ```patternfly/react-token``` variable.

```import { global_Color_200 as color200 } from '@patternfly/react-tokens';```

Reply:
fixed in https://github.com/openshift/console/pull/2496/commits/7c710a2c732716fb8b688fcb97b5f3a19a9472d2